### PR TITLE
Add posibility to use external configuration

### PR DESCRIPTION
**Pull request by KseniiaSych**

### CODE REVIEW

File: user_agent/configuration.py [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+                    SchemaError("Too much values in {}".format(browser))
+                ])
+
+    # MACOSX_CHROME_BUILD_RANGE versions should be present in OS_PLATFORM
+    mac_platforms = list(
+        map(lambda platform_string: platform_string.split('OS X ')[1],
+            external_config['OS_PLATFORM']['mac']))
+    for version_key in external_config['MACOSX_CHROME_BUILD_RANGE'].keys():
+        try:
+            assert version_key in mac_platforms
```
Comment (lorien) [inline user_agent/configuration.py]: Why so complex? It could be rewritten just as
```
if version_key not in mac_platforms:
    raise Something(...)
```

**Comment by lorien** [comment]
I've taken a quick look on the pull request. Looks fine except I do not see tests. All new functionality have to be covered by tests.

File: user_agent/configuration.py (hunk 1 above)
Comment (KseniiaSych) [reply user_agent/configuration.py]: Yes, you're right I've overdone this part

**Comment by lorien** [comment]
Ok, I am back to work on this project.
Thank you for you work.
Sorry, I can't accept this pull request. It complicates my library more than I want. Also I am going to refactor it to use more simple data generation. After refactoring there will be clear how to implement loading user-customized data.

**Pull request closed without merging**